uk.co.compucorp.civicrm.hrcore/js/src/hrcore/app.js

```diff
@@ -1,37 +1,8 @@
-/* global dataLayer */
 /* eslint-env amd */
 
-(function (window, CRM, dataLayer) {
-  define(function () {
-    'use strict';
-
-    trackContactTabVirtualPageviews();
-
-    /**
-     * Sends a virtual pageview to Google Analytics whenever the user clicks on
-     * any of the tabs of the Contact Summary page
-     *
-     * The path of the pageview is the Contact Summary page path + the selectedChild
-     * param's value of the selected tab, thus simulating the user landing
-     * directly on the tab
-     */
-    function trackContactTabVirtualPageviews () {
-      var contactPagePath;
-
-      if (!dataLayer) {
-        return;
-      }
-
-      contactPagePath = window.location.pathname + '?reset=1&cid=' + CRM.contactId;
-
-      CRM.$('#mainTabContainer').on('tabsactivate', function (event, ui) {
-        var tabName = ui.newTab[0].id.replace('tab_', '');
-
-        dataLayer.push({
-          event: 'virtual-pageview',
-          virtualPageviewPath: contactPagePath + '&selectedChild=' + tabName
-        });
-      });
-    }
-  });
-}(window, CRM, (typeof dataLayer !== 'undefined' ? dataLayer : null)));
+define([
+  'hrcore/analytics/page-tracker',
+  'hrcore/helpers/help-icon-injector'
+], function (trackContactTabVirtualPageviews) {
+  trackContactTabVirtualPageviews();
```

Comment:
sorry to say but now you gotta test if the page view tracking still works

Reply:
I tested if the func is called with a proper params. And will test again after the latest changes.